### PR TITLE
add nquarder to README.md

### PR DESCRIPTION
**Pull request by nquarder**

### CODE REVIEW

**Review by eculler — CHANGES_REQUESTED** [review]
The link doesn't work! Please fix.

**Comment by nquarder** [comment]
@eculler thanks for catching this mistake! I think I've fixed it now. -NQ

**Review by eculler — DISMISSED** [review]
w00t!

**Review by cmq879 — APPROVED** [review]

**Comment by nquarder** [comment]
adding map of CR IA to posts/01-first-map directory cc: @atticusbaker & @npwelsh can you please review when you get a second? thanks! -NQ

File: posts/01-first-map/nquarder/cedar-rapids.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Kirkwood Community College (Cedar Rapids, IA)
+
+I went to Kirkwood CC in CR, IA 1999-2001. On July 1, 1966, Kirkwood was officially established, originally called "Area X Community College." The college immediately assumed responsibility for the federally-funded vocational/technical programs the Cedar Rapids Community School District had provided since 1964. [Source](https://creditcatalog.kirkwood.edu/kirkwood-history-accreditation/#:~:text=On%20July%201%2C%201966%2C%20Kirkwood,District%20had%20provided%20since%201964.)
```
Comment (npwelsh) [inline posts/01-first-map/nquarder/cedar-rapids.md:3]: Maybe change spell out CC and CR into Community College and Cedar Rapids respectively?  Other than that look great!

**Review by npwelsh — COMMENTED** [review]
Hi Nate, looks good! I like how you put two different polygons on the map looks great.  In cedar-rapids.md, have a small suggestion on line 3.  Other than that looks great!

**Pull request merged**